.flake8

```diff
@@ -66,6 +66,10 @@ exclude =
 
 format = spack
 
+# flake8-import-order settings
+application-import-names = llnl,spack
+import-order-style = google
```

Comment:
```suggestion
import-order-style = pycharm
```

Reply:
I like `pycharm` mainly because I put the `import` statements before the `from X import Y` statements -- it's more readable.  Also JetBrains and @alalazo usually have good taste about these things 😄